File: HotDrink.py
from Drink import Drink


class HotDrink(Drink):
    def __init__(self, name, value, temperature):
        # super().__init__(name, value)
        self.__name = name
        self.__value = value
        self.__temperature = temperature

    def getname(self):
        return self.__name

    def setname(self, name):
        self.__name = name

    def getvalue(self):
        return self.__value

    def setvalue(self, value):
        self.__value = value

    def getTemperature(self):
        return self.__temperature

    def setTemperature(self, temperature):
        self.__temperature = temperature

    def __str__(self):
        return f"Название: {self.__name}  Объём: {self.__value}  Температура: {self.__temperature}"

File: HotDrink.java
package lesson_4;


public class HotDrink {
    private int temperature;
    private float value;
    private String name;

    HotDrink(String name, float value, int temperature) {
        this.name = name;
        this.value = value;
        this.temperature = temperature;
    }

    public int getTemperature() {
        return temperature;
    }

    public void setTemperature(int temperature) {
        this.temperature = temperature;
    }

    public float getValue() {
        return value;
    }

    public void setValue(float value) {
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String toString() {
        return String.format("%s  %.1f  %d", name, value, temperature);
    }

}